src/dawn/SIR/Statement.h

```diff
@@ -15,6 +15,7 @@
 #ifndef DAWN_SIR_STATEMENT_H
 #define DAWN_SIR_STATEMENT_H
 
+#include "dawn/SIR/ASTStmt.h"
```

Comment:
should be a fwd-declaration

Reply:
need the definition due to https://github.com/MeteoSwiss-APN/dawn/blob/931abab96bcd90be2850782045135d5ab195d73b/src/dawn/SIR/Statement.h#L48

Reply:
statement will be gone shortly anyways. This is fine